lib/src/features/line_renderer.dart

```diff
@@ -47,7 +47,13 @@ class LineRenderer extends FeatureRenderer {
     effectivePaint.strokeWidth =
         context.tileSpaceMapper.widthFromPixelToTile(strokeWidth);
 
-    final lines = feature.paths;
+    var dashLengths = style.linePaint?.dashArray ?? [];
+    // map dash lengths to correct tile unit
+    dashLengths = dashLengths.map((e) =>
+        context.tileSpaceMapper.widthFromPixelToTile(e.toDouble())
```

Comment:
I missed this before. There's a good chance (e.g. as we're zooming) that we'll get a whole bunch of variations on dashLengths. Now I'm not so sure that we should be caching these paths, since we could end up with a lot of them. What do you think?